### PR TITLE
70 what tests do we need

### PR DESCRIPTION
**Pull request by marcosfrenkel**
Added tests for almost all of quAPL.

Tests for the show module are missing, since I don't believe this module is ready for them. This is because they lack any sort of tools for multiple qubit vector states. There are some functions for registers, but if you are working with register you already have the qubits separated which I am not sure if that is what the show module should be.

### CODE REVIEW

File: Tests/test_thread_003.aplf [hunk 1]
```diff
@@ -0,0 +1,11 @@
+test_thread_003 ← {
+    correct_result ← 16 1 ⍴ (16/0.25)
+    a ← 2 1 ⍴ (1÷(2*0.5)) (1÷(2*0.5))
```
Comment (nunezco2) [inline Tests/test_thread_003.aplf:3]: Use an additional variable to avoid repeating this regular pattern.

File: Tests/test_thread_gate_001.aplf [hunk 2]
```diff
@@ -0,0 +1,8 @@
+test_thread_gate_001 ← {
+    correct_result ← 4 4⍴(0 (0.5*0.5) 0 (0.5*0.5) (0.5*0.5) 0 (0.5*0.5) 0 0 (0.5*0.5) 0 (¯1×0.5*0.5) (0.5*0.5) 0 (¯1×0.5*0.5) 0)
```
Comment (nunezco2) [inline Tests/test_thread_gate_001.aplf:2]: Same here

File: Tests/test_unkpr_002.aplf [hunk 3]
```diff
@@ -0,0 +1,9 @@
+test_unkpr_002 ← {
+    a ← 2 1 ⍴ (1÷(2*0.5)) (1÷(2*0.5))
```
Comment (nunezco2) [inline Tests/test_unkpr_002.aplf:2]: Same here

File: Tests/test_unthread_vs_001.aplf [hunk 4]
```diff
@@ -0,0 +1,12 @@
+test_unthread_vs_001 ← {
+    a ← 2 1 ⍴ 1 0 
+    b ← 2 1 ⍴ 1 0 
+    c ← 2 1 ⍴ 1 0
```
Comment (nunezco2) [inline Tests/test_unthread_vs_001.aplf:4]: Identical, use only one

File: Tests/test_unthread_vs_002.aplf [hunk 5]
```diff
@@ -0,0 +1,10 @@
+test_unthread_vs_002 ← {
+    a ← 2 1 ⍴ (1÷(2*0.5)) (1÷(2*0.5))
```
Comment (nunezco2) [inline Tests/test_unthread_vs_002.aplf:2]: Same here, single value then use variable

File: Tests/test_unthread_vs_003.aplf [hunk 6]
```diff
@@ -0,0 +1,11 @@
+test_unthread_vs_003 ← {
+    a ← 2 1 ⍴ (1÷(2*0.5)) (1÷(2*0.5))
```
Comment (nunezco2) [inline Tests/test_unthread_vs_003.aplf:2]: Use single variable, use one larger assignment only later

File: failed_tests/test_qx_value_002.aplf [hunk 7]
```diff
@@ -0,0 +1,8 @@
+ ⍝ Fails (I think) because the qx product is defined backwards, x.+ is not the same as +.x.
+ test_qx_value_002 ← {
+     correct_value ← 2 1 ⍴ ((1÷(82*0.5))(9÷(82*0.5)))
```
Comment (nunezco2) [inline failed_tests/test_qx_value_002.aplf:3]: Same here

**Review by nunezco2 — CHANGES_REQUESTED** [review]

**Comment by marcosfrenkel** [comment]
@nunezco2  I cleaned up all of the inconsistent whitespace between all the files. I have also removed any repeated pattern. Let me know if you like the way I fixed it (wasn't entirely sure if that is what you meant)

**Review by nunezco2 — APPROVED** [review]

**Pull request merged**